### DISCUSSION TITLE
Color Fill Greyed Out

### DISCUSSION THREAD

**Discussion by chrsrns** [phase1geo/Minder#435] [Q&A]
Hi.

On my Flatpak installation, the Color Fill switch is greyed out. Am I missing something?

![Screenshot](https://user-images.githubusercontent.com/43345572/144574319-4f6528a9-f3ca-456b-9574-0c6aaeb22aeb.png)

**Comment by phase1geo** [comment]
Some node borders are not "fillable", including the one selected in your screenshot.

**Reply by phase1geo** [reply]
The first three are not; the last three are.
